CMakeLists.txt

```diff
@@ -92,6 +92,17 @@ list(APPEND CMAKE_MODULE_PATH
   ${CMAKE_CURRENT_LIST_DIR}/absl/copts
 )
 
+if(${CMAKE_VERSION} VERSION_GREATER 3.15 OR ${CMAKE_VERSION} VERSION_EQUAL 3.15)
```

Comment:
Our CMake minimum version was recently raised to 3.16, so I think this can be left out.